samples/Tutorials/CSharpBeginner/CSharpBeginner/Resources/readme.txt

```diff
@@ -0,0 +1,11 @@
+# Help
+There are Videos and a manual that go with this tutorial:
+ - https://www.youtube.com/watch?v=Z2kUQhSmdr0&list=PLRZx2y7uC8mNySUMfOQf-TLNVnnHkLfPi
+ - https://doc.stride3d.net/latest/en/tutorials/csharpbeginner/index.html
+ - Feel free to ask any questions in the Dicsord as well or use the Github issues board to notify of bugs
```

Comment:
Github should be GitHub, and typo in Dicsord 🤣

Reply:
good call lol should be fixed now